src/components/containers/Panel.js

```diff
@@ -1,21 +1,45 @@
 import React, {Component} from 'react';
 import PropTypes from 'prop-types';
-import {bem} from '../../lib';
+import {bem} from 'lib';
+
+class PanelHeader extends Component {
+  render() {
+    const {children, action} = this.props;
+
+    return (
+      <div className={bem('panel', 'header')}>
+        <div className={bem('panel', 'header__content')}>{children}</div>
+        {action && (
+          <div className={bem('panel', 'header__actions')}>{action}</div>
+        )}
+      </div>
+    );
+  }
+}
 
 class Panel extends Component {
   render() {
-    if (this.props.visible) {
-      return <div className={bem('panel')}>{this.props.children}</div>;
+    const {visible, children} = this.props;
+
+    if (visible) {
+      return <div className={bem('panel')}>{children}</div>;
     }
-    return <div />;
+    return null;
   }
 }
 
+Panel.Header = PanelHeader;
+
```

Comment:
why not just export PanelHeader itself? we don't use this convention anywhere else.
some files have multiple exports, we export each individually, and import with `{}`

Reply:
I did it mostly for backwards compatibility and it's a convention I recently started following for creating compound components. For example, I can go into the Batch app and just use `<Panel.Header>...</Panel.Header>` without having to import an additional thing. I didn't want to change the convention of panel from `import Panel from...` to `import {Panel, PanelHeader} from...` as that would break every instance of `import Panel`. We could do `import Panel, {PanelHeader} from...`.

Reply:
> We could do import Panel, {PanelHeader} from....
Yeah, it's the one we use throughout our code I think